### DISCUSSION TITLE
add rope version of alucard24 in SwarmUI

### DISCUSSION THREAD

**Discussion by ivaxsirc** [mcmonkeyprojects/SwarmUI#53] [Ideas]
The version of Alucard24 supports GPEN1024 definitely better and Landmarks Detection, The combination of ic-light with reactor gives very realistic deepfakes. I have tried it with forge and the result is very good, it would be very interesting if SwarmUI implemented this option with the best deepfaker rope pearl alucard24 version 
Thank you

**Comment by ZeroCool22** [comment]
Better than Reactor?
